src/main/java/frc/robot/util/CommandController.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.util;
+
+import frc.robot.Constants;
+import frc.robot.arm.Arm;
+
+public class CommandController {
```

Comment:
This name doesn't quite describhe the prupose of this class. It doesn't control the commands, commands use these methods to determine what state the robot is in

Names I like (take your pick)

`RobotContext`
`RobotState`
`InterlockState`
`RobotInfo`